<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comunicación entre PC - Mbed</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>871</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>691</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Flujo de Datos</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>661</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>162</width>
      <height>97</height>
     </rect>
    </property>
    <property name="title">
     <string>Comunicaciones</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBox">
       <property name="currentText">
        <string>Lista de Comandos</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Lista de Comandos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ALIVE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GET_LEDS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SET_LEDS</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButtonEnviar">
       <property name="text">
        <string>Enviar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>871</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Leds</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>20</y>
       <width>841</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxLed1">
        <property name="text">
         <string>LED_RED</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxLed2">
        <property name="text">
         <string>LEG_BLUE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxLed3">
        <property name="text">
         <string>LED_YELLOW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxLed4">
        <property name="text">
         <string>LED_GREEN</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>103</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonOff">
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButtonOn">
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuComunicaci_n">
     <property name="title">
      <string>Comunicación</string>
     </property>
     <addaction name="actionEscaneo_de_Puertos"/>
     <addaction name="actionDesconectar"/>
     <addaction name="actionConectar"/>
    </widget>
    <addaction name="menuComunicaci_n"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionEscaneo_de_Puertos">
   <property name="text">
    <string>Escaneo de Puertos</string>
   </property>
  </action>
  <action name="actionDesconectar">
   <property name="text">
    <string>Desconectar</string>
   </property>
   <property name="toolTip">
    <string>Desconectar</string>
   </property>
  </action>
  <action name="actionConectar">
   <property name="text">
    <string>Conectar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
